net/netcheck/netcheck.go

```diff
@@ -1321,9 +1321,9 @@ func (c *Client) measureAllICMPLatency(ctx context.Context, rs *reportState, nee
 	ctx, done := context.WithTimeout(ctx, icmpProbeTimeout)
 	defer done()
 
-	p, err := ping.New(ctx, c.logf, c.NetMon)
-	if err != nil {
-		return err
+	p := ping.New(ctx, c.logf, c.NetMon)
+	if p == nil {
```

Comment:
you can drop the nil check if you want, a panic is fine in this exceptional case